corehq/db/config.py

```diff
@@ -0,0 +1,31 @@
+import yaml
+from copy import copy
+
+from dimagi.utils.decorators.memoized import memoized
+
+
+class DBConfig(object):
+    def __init__(self, config_path):
+        """
+        Takes a path to a yaml config file. See corehq/db/config.yml
+        """
+        self.config_path = config_path
+
+    @property
+    @memoized
+    def config(self):
+        with open(self.config_path, 'r') as f:
+            return yaml.load(f.read())
+
+    def to_django_config(self):
+        django_config = {}
+        common = self.config.get('common', {})
+
+        for database_name, database_config in self.config['databases'].items():
+            django_config[database_name] = copy(common)
```

Comment:
(no action required) you can do the same thing with `common.copy()` if it's always a dict (no need to import copy)